src/boxes/file_level_boxes.cpp

```diff
@@ -24,6 +25,11 @@ LovokStatusCode ParseMoov(FileWrapper *fileWrapper, uint64_t length, uint64_t by
             result = ParseMoovMeta(fileWrapper, header.size, byteOffset);
         } else if (!strcmp(header.name, "mvex")) {
             result = ParseMvex(fileWrapper, header.size, byteOffset);
+        } else if (!strcmp(header.name, "udta")) {
+            result = ParseMoovUdta(fileWrapper, header.size, byteOffset);
+        } else {
+            std::cout << "Unknown box name in ParseMoov: ";

```

Comment:
Can you change these so they are not referencing the function name? Could this read like `Encountered unknown box when parsing moov box`